content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -71,35 +81,28 @@ This analysis uses the same logic and error messages as when using `istioctl ana
 
 For example. if you have a misconfigured gateway on your "ratings" virtual service, running `kubectl get virtualservice ratings` would give you something like:
 
-{{< text yaml >}}
-apiVersion: networking.istio.io/v1alpha3
+{{< text syntax=yaml snip_id=vs_yaml_with_status >}}
+apiVersion: networking.istio.io/v1beta1
 kind: VirtualService
 metadata:
-  annotations:
-    kubectl.kubernetes.io/last-applied-configuration: |
-      {"apiVersion":"networking.istio.io/v1alpha3","kind":"VirtualService","metadata":{"annotations":{},"name":"ratings","namespace":"default"},"spec":{"hosts":["ratings"],"http":[{"route":[{"destination":{"host":"ratings","subset":"v1"}}]}]}}
-  creationTimestamp: "2019-09-04T17:31:46Z"
-  generation: 11
+...
   name: ratings
   namespace: default
-  resourceVersion: "12760039"
-  selfLink: /apis/networking.istio.io/v1alpha3/namespaces/default/virtualservices/ratings
-  uid: dec86702-cf39-11e9-b803-42010a8a014a
+...
 spec:
   gateways:
   - bogus-gateway
   hosts:
   - ratings
-  http:
-  - route:
-    - destination:
-        host: ratings
-        subset: v1
+...
```

Comment:
Do you really need this many `...` in the example? It looks kind of ugly in the doc. Can't you cut it down to a couple and still get the point across?

Reply:
Maybe I don't understand `...` and elided.  For the user I thought it means "unimportant stuff" and for the recognizer I thought it means "*" (wildcard).  So for example I didn't want to show that there might be a second reported problem, so I used ... there.

Reply:
You're right that `...` is a wildcard for testing and that in docs it's used to elide "unimportant stuff", but what you are doing is complicating the doc by trying to make the example a fully correct regular expression, which isn't necessary or helpful. In docs, you really just want to show a simple specific example, and only elide things that are noise (e.g., metadata or run specific values like date/time). The "unimportant stuff" you should elide is stuff in the original example that isn't needed to support what the document is trying to explain with the example. The original example didn't need to indicate the possible presence of other unrelated stuff that isn't relevant to the discussion at hand, so the elided one shouldn't either. Elision in docs is used to simplify, not to produce a generally correct regular expression.

Reply:
I would leave the invalid specification in the doc, so it's clear what the error message is referring to.
```
kind: VirtualService
...
spec:
  gateways:
  - bogus-gateway
  hosts:
  - ratings
  http:
  - route:
    - destination:
        host: ratings
        subset: v1
status:
  validationMessages:
...
  - code: IST0101
    documentation_url: https://istio.io/docs/reference/config/analysis/IST0101?ref=status-controller
    level: Error
    message: 'Referenced gateway not found: "bogus-gateway"'
...
```